_posts/2020-03-24-dask-developer-workshop.markdown

```diff
@@ -0,0 +1,85 @@
+---
+layout: single
+title:  "Tales from D.C.: Dask Developer Workshop 2020"
+date:   2020-03-24 20:00:00 +0100
+tags: technology python data-engineering dask distributed
+header:
+  overlay_image: assets/images/tech_gear_banner.jpg
+  overlay_filter: 0.2
+  show_overlay_excerpt: false
+author: Lucas Rademaker
+author_profile: true
+---
+# Tales from D.C.: Dask Developer Workshop 2020
+
+## Setting
+Back in January this year, [Dask](https://dask.org/) community members received an invite for a workshop in Washington DC with the goal of "connecting advanced users with developers".
+
+## The workshop
+
+The schedule of the workshop alone was quite satisfying to me: a number of talks, involving users and developers of Dask and related projects (e.g. NumPy, xarray, Arrow), combined with working sessions, wherein one had the opportunity to discuss or code with other attendees.
+
+What follows is an outline of my experience at the workshop.
+### Talks
+##### ETL Pipelines
+ - Blue Yonder and [Kartothek](https://github.com/JDASoftwareGroup/kartothek).
+ 
+    My colleague Florian Jetter was the first to present. He outlined our use case of executing Dask pipelines coupled with tight SLA's (Service Level Agreements), which obliges us to make sure that our Dask pipelines finish successfully each run, as usually we do not have time to re-schedule. He also talked about some of the issues we had faced in the past and others which we are facing in the present (mostly related to stability).
```

Comment:
I would be interested in more details here, like what exactly is our use case for these dask pipelines? Which issues are we facing?
Obviously, we don't need a full summary of the talk. But just mentioning that we "faced issues" without actually naming what thise issues were feels a bit unsatisfactory.